rq_dashboard/web.py

```diff
@@ -180,7 +181,8 @@ def overview(queue_name, page):
         queue=queue,
         page=page,
         queues=get_all_queues(),
-        rq_url_prefix=url_for('.overview')
+        rq_url_prefix=url_for('.overview'),
+        rqdVersion=VERSION
```

Comment:
Please prefer more human-readable variable names. Like rq_dashboard_version.
Also, please use snake_case for Python code and try to match style of existing code.

Reply:
And remember the RQ version :)